### ISSUE TITLE
RESTful API / login not returning "Set-Cookie: JSESSIONID=xxxxxx"

### ISSUE THREAD

**Issue by jolypas** [vitalidze/traccar-web#1164]
Hello

I have two instances of web-traccar (one stable, one for test) which are in fact clone of Virtual VM servers.

I'm using rest API/login  from another process without issue on one server.
on the second one , depiste returned code is "HTTP/1.1 200 OK", the line 
_< Set-Cookie: JSESSIONID=xxxxxxxxxxxxxxxxxxxxxxx;Path=/_
is missing so I cannot use the other services.

all other data returned are OK.

Any idea of the root cause for that ? (both of them are https and the SSL certificates are OK)

Thanks

**Comment by vitalidze** [comment]
Maybe your reverse proxy hides the `set-cookie` header? Have you tried accessing traccar web server directly (i.e. through 8082 port)?

**Comment by jolypas** [comment]
Hello 

I have no reverse proxy installed. 
I have the same result with http and 8082 (no cookies in header) 

Regards 

Le 9 août 2017 21:24, à 21:24, Vitaly Litvak <notifications@github.com> a écrit:

**Comment by vitalidze** [comment]
Okay, does it work in web-browser? Can you log in to your web UI?

**Comment by jolypas** [comment]
yes the desktop HMI (web browser)  works very fine in both https and http
with port 8082



2017-08-10 10:32 GMT+02:00 Vitaly Litvak <notifications@github.com>:

**Comment by vitalidze** [comment]
The thing is that web browser is using exactly same cookie. I guess there is some problem in your application, which consumes the API.

**Comment by jolypas** [comment]
the problem is seen also using CURL , so I don't thing about the
application (the same application is using the other server without issue)

2017-08-10 15:25 GMT+02:00 Vitaly Litvak <notifications@github.com>:

**Comment by vitalidze** [comment]
Maybe there is some cache in your cURL, which uses cookies. If you can access it from web browser, then this problem is not with web server. You can check headers in web browser in Developer Tools.
